# `python:3.4.5-alpine`

## Docker Metadata

- Image ID: `sha256:0eb0091592b3d8aab929e19041330d307e0e3302cf58ae8753276a2860c45037`
- Created: `2016-12-27T21:39:19.757995673Z`
- Arch: `linux`/`amd64`
- Command: `["python3"]`
- Environment:
  - `PATH=/usr/local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin`
  - `LANG=C.UTF-8`
  - `GPG_KEY=97FC712E4C024BBEA48A61ED3A5CA953F73C700D`
  - `PYTHON_VERSION=3.4.5`
  - `PYTHON_PIP_VERSION=9.0.1`
